packages/nimble/R/cppDefs_cppProject.R

```diff
@@ -206,19 +206,14 @@ cppProjectClass <- setRefClass('cppProjectClass',
                                            "}")
                                        writeLines(contentLines, con = dynamicRegistrationsCppName)
                                    },
-                                   compileDynamicRegistrations = function(showCompilerOutput = nimbleOptions('showCompilerOutput')) {
-                                       timeStamp <- format(Sys.time(), "%m_%d_%H_%M_%S")
-                                       
-                                       dynamicRegistrationsDllName <- paste0("dynamicRegistrations_", timeStamp)
-                                       dynamicRegistrationsCppName <- paste0(dynamicRegistrationsDllName, ".cpp")
-                                       
-                                       writeDynamicRegistrationsDotCpp(dynamicRegistrationsCppName, dynamicRegistrationsDllName)
-                                       ssDllName <- file.path(dirName, paste0(dynamicRegistrationsDllName, .Platform$dynlib.ext))
-                                       ssdSHLIBcmd <- paste(file.path(R.home('bin'), 'R'), 'CMD SHLIB', dynamicRegistrationsCppName, '-o', basename(ssDllName))
+                                   compileStaticCode = function(dllName, cppName) {
```

Comment:
@perrydv Could you review this file for sanity? I've tried to follow your suggestion.